_events/CUBiDA_colloquium_Erlangen_Introduction_to_Galaxy.md

```diff
@@ -0,0 +1,24 @@
+---
+site: [Universitätklinikum Erlangen, CUBiDA, FAU, Erlangen, Germany]
+tags:
+- Talk
+- Tutorial
+title: "Introduction to Galaxy - a web-based platform for bioinformatics analysis"
+starts: 2021-06-23
+ends: 2021-06-23
+organiser: Pilot Core Unit Bioinformatics, Data Integration and Analysis, Universitätklinikum Erlangen
+location: online (https://uk-erlangen.webex.com/uk-erlangen/j.php?MTID=mfa8088026974da77de04e981fb3b7abd)
+supporters: 
+ - galaxy-europe
+ - galaxy team freiburg
+ - Life@FAU
```

Comment:
@PGbioinf do you think we should/could add MIRACUM here?

Reply:
yes, thank you so much Bjoern!

Reply:
I'll do it :)

Reply:
> 
> 
> I'll do it :)

Thank you so much!